Common/Sources/View/Error/ErrorView.swift

```diff
@@ -22,13 +22,13 @@ import UIKit
 
 /// The class to handle UI errors
 public final class ErrorView {
+    private let errorTypeApi = "API"
     private let errorTypeConnection = "CONNECTION"
-
+    private let hyperwalletInsights: HyperwalletInsightsProtocol
```

Comment:
why do you need this? Why not directly use HyperwalletInsights.shared.trackError(...)?

Reply:
I didn't add this - simply moved this as per ordering conventions - Yes, I don't see why this would be required.
Looking at https://github.com/hyperwallet/hyperwallet-ios-ui-sdk/pull/170 - looks like it was added for testing and removed in a later commit.